packages/gafl-webapp-service/src/pages/layout/layout.njk

```diff
@@ -23,7 +23,7 @@
             homepageUrl: "https://www.gov.uk/",
             serviceUrl: "/",
             containerClasses: "govuk-width-container",
-            serviceName: "Get a rod fishing license" if not TELESALES else "Get a rod fishing license (TELESALES)"
+            serviceName: "Get a rod fishing licence" if not TELESALES else "Get a rod fishing licence (TELESALES)"
```

Comment:
The old service was called "Get a fishing licence" not "Get a rod fishing licence"

Reply:
TBC

Reply:
👍  lets wait to see what Claire decides